airflow/providers/docker/operators/docker_swarm.py

```diff
@@ -86,8 +86,19 @@ class DockerSwarmOperator(DockerOperator):
     :param mode: Indicate whether a service should be deployed as a replicated or global service,
         and associated parameters
     :param networks: List of network names or IDs or NetworkAttachmentConfig to attach the service to.
-    :param placement: Placement instructions for the scheduler. If a list is passed instead,
-        it is assumed to be a list of constraints as part of a Placement object.
+        NetworkAttachmentConfig object accepts following params:
```

Comment:
We already have a link to Docker API reference.
<img width="932" alt="Screenshot 2022-03-13 at 15 29 50" src="https://user-images.githubusercontent.com/12058428/158064360-0a96c238-527d-4a15-a54f-d66cd705d4d8.png">
I am not sure if we need to duplicate this documentation as it can be maintenance burden

Reply:
yes, it makes sense. I have raised this PR in response to this issue #15974
It looked like the constraints and network option is overlooked. therefore I have added this to the documentation. What do you suggest wrt this issue? @mik-laj

Reply:
I guess we can close the ticket with the comment that we can't reproduce the problem because we have the documentation, and it's only natural for someone to miss it sometimes.

Reply:
hmm ok @mik-laj. That makes sense.